src/main/java/com/enterprise/fundraisermanager/dao/FundraiserSQLDAO.java

```diff
@@ -0,0 +1,35 @@
+package com.enterprise.fundraisermanager.dao;
+
+import com.enterprise.fundraisermanager.dto.Fundraiser;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+
+@Repository("fundraiserDAO")
+public class FundraiserSQLDAO implements IFundraiserDAO {
+
+    @Autowired
+    FundraiserRepository fundraiserRepository;
+
+    @Override
+    public Fundraiser save(Fundraiser fundraiser) throws Exception {
+        Fundraiser createdFundraiser = fundraiserRepository.save(fundraiser);
+        return createdFundraiser;
+    }
+
+    @Override
+    public List<Fundraiser> fetchAll() {
+        return null;
+    }
+
+    @Override
+    public Fundraiser fetch(int id) {
+        return null;
+    }
+
+    @Override
+    public void delete(int id) {
+
+    }
```

Comment:
This appears to be an incomplete feature.
Only features that meet the team's Definition of Done should be merged to master.
Otherwise, it is clutter, which is technical debt